examples/lighting-app/silabs/src/LightingManager.cpp

```diff
@@ -307,3 +307,21 @@ void LightingManager::OnTriggerOffWithEffect(OnOffEffect * effect)
     LightMgr().mOffEffectArmed = true;
     LightMgr().StartTimer(offEffectDuration);
 }
+#ifdef SL_MATTER_RGB_LED_ENABLED
+bool LightingManager::InitiateLightAction(int32_t aActor, Action_t aAction, uint16_t size, uint32_t * value)
+{
+    bool action_initiated = false;
+    switch (aAction)
+    {
+    case COLOR_ACTION_HSV:
+    case COLOR_ACTION_CT:
+    case COLOR_ACTION_XY:
+        action_initiated = true;
+        AppTask::GetAppTask().PostLightControlActionRequest(aActor, aAction, value);
```

Comment:
if they all lead to the same call, why use a switch case? Why to use a VerifyOrReturnError (false) to check for a known action
else call 
```
AppTask::GetAppTask().PostLightControlActionRequest(aActor, aAction, value);
return true;
```

Reply:
Changed to `VerifyOrReturnError`